src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -173,21 +173,38 @@ the main body of the message as well as a quote.
 		</div>
 
 		<!-- Message actions -->
-		<MessageButtonsBar v-if="showMessageButtonsBar"
-			ref="messageButtonsBar"
-			:is-action-menu-open.sync="isActionMenuOpen"
-			:is-emoji-picker-open.sync="isEmojiPickerOpen"
-			:is-reactions-menu-open.sync="isReactionsMenuOpen"
-			:is-forwarder-open.sync="isForwarderOpen"
-			:message-api-data="messageApiData"
-			:message-object="messageObject"
-			:is-last-read="isLastReadMessage"
-			:can-react="canReact"
-			v-bind="$props"
-			:previous-message-id="previousMessageId"
-			:participant="participant"
-			:style="messageButtonsBarStyle"
-			@delete="handleDelete" />
+		<div class="message-body__scroll">
+			<MessageButtonsBar v-if="showMessageButtonsBar"
+				ref="messageButtonsBar"
+				:is-action-menu-open.sync="isActionMenuOpen"
+				:is-emoji-picker-open.sync="isEmojiPickerOpen"
+				:is-reactions-menu-open.sync="isReactionsMenuOpen"
+				:is-forwarder-open.sync="isForwarderOpen"
+				:message-api-data="messageApiData"
+				:message-object="messageObject"
+				:is-last-read="isLastReadMessage"
+				:can-react="canReact"
+				v-bind="$props"
+				:previous-message-id="previousMessageId"
+				:participant="participant"
+				@delete="handleDelete">
+				<NcActionButton :aria-label="messageDate">
+					<template #icon>
+						<CheckAll v-if="showCommonReadIcon"
+							:title="commonReadIconTooltip"
+							:aria-label="commonReadIconTooltip"
+							:size="16" />
+						<Check v-else-if="showSentIcon"
+							:title="sentIconTooltip"
+							:aria-label="sentIconTooltip"
+							:size="16" />
+						<ClockOutline v-else :size="16" />
+					</template>
+					{{ messageDateTime }}
+				</NcActionButton>
+			</MessageButtonsBar>
```

Comment:
Feel inconsistent that all MessageButtonBar's NcActionButtons are a part of the component while this new NcActionButton is passed with a slot.

Is this slot supposed to have anything else except this one button? Or is there something about this NcActionButton why it should be passed into the slot, unlike all the other buttons?

Reply:
> It's shown outside and inside the component, could also duplicate it

What do you mean?